Makefile

```diff
@@ -38,9 +42,28 @@ test-unit:
 test-functional:
 	pipenv run ./scripts/run_tests_functional.sh
 
+test-functional-headless:
+	EQ_RUN_FUNCTIONAL_TESTS_HEADLESS='True' pipenv run ./scripts/run_tests_functional.sh
+
+test-single-functional:
+	yarn test_functional --spec ./tests/functional/spec/$(SPEC)
+
+test-suite-functional:
+	yarn test_functional --suite $(SUITE)
+
+lint-js:
+	yarn lint
+
+lint-test-js: lint-js test-functional
+
+test-headless-lint-js: lint-js test-functional-headless
```

Comment:
We need to be careful from making the make commands too verbose and adding combinations we won't use.
Can devs verify that all the commands we have are all useful.

Reply:
The command on 59 was specified on the ticket itself, however, when running both JS test + lint, I thought it would be appropriate to provide the option to allow the dev to run JS test + lint headless. If the name is an issue, I can think of a different one.

Reply:
I think if we do `lint-test-js` it makes sense to also have a headless version, I'd make the names more consistent though, like `lint-test-js` and `lint-test-headless-js`. That said, I probably won't use these two